src/robusta/integrations/kubernetes/autogenerated/events.py

```diff
@@ -55,6 +55,7 @@
     RobustaDeployment,
     RobustaJob,
     RobustaPod,
+    DeploymentConfig,
```

Comment:
This file is auto generated
Need to think how to include the DeploymentConfig, which is special because it's not a hikaru builtin resource